WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsParentFragment.kt

```diff
@@ -0,0 +1,107 @@
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.google.android.material.tabs.TabLayout.OnTabSelectedListener
+import com.google.android.material.tabs.TabLayout.Tab
+import com.google.android.material.tabs.TabLayoutMediator
+import org.wordpress.android.R
+import org.wordpress.android.databinding.BloggingPromptsParentFragmentBinding
+import org.wordpress.android.ui.bloggingprompts.PromptSection.ALL
+import org.wordpress.android.ui.bloggingprompts.PromptSection.ANSWERED
+import org.wordpress.android.ui.bloggingprompts.PromptSection.NOT_ANSWERED
+
+class BloggingPromptsParentFragment : Fragment() {
+    private lateinit var binding: BloggingPromptsParentFragmentBinding
+
+    private val viewModel: BloggingPromptsParentViewModel by activityViewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        binding = BloggingPromptsParentFragmentBinding.inflate(inflater)
+        setupToolbar(binding)
+        setupTabLayout(binding)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        viewModel.onOpen()
+    }
+
+    private fun setupToolbar(binding: BloggingPromptsParentFragmentBinding) {
+        with(binding) {
+            with(activity as AppCompatActivity) {
+                setSupportActionBar(toolbar)
+
+                title = getString(R.string.blogging_prompts_title)
+                supportActionBar?.setDisplayShowTitleEnabled(true)
+                supportActionBar?.setDisplayHomeAsUpEnabled(true)
+            }
+        }
+    }
+
+    private fun setupTabLayout(binding: BloggingPromptsParentFragmentBinding) {
+        with(binding) {
+            val adapter = BloggingPromptsPagerAdapter(this@BloggingPromptsParentFragment)
+            promptPager.adapter = adapter
+            TabLayoutMediator(tabLayout, promptPager) { tab, position ->
+                tab.text = adapter.getTabTitle(position)
+            }.attach()
+            tabLayout.addOnTabSelectedListener(SelectedTabListener(viewModel))
+        }
+    }
+
+    companion object {
+        const val LIST_TYPE = "type_key"
+
+        fun newInstance(section: PromptSection): BloggingPromptsParentFragment {
+            val fragment = BloggingPromptsParentFragment()
+            val bundle = Bundle()
+            bundle.putSerializable(LIST_TYPE, section)
+            fragment.arguments = bundle
+            return fragment
+        }
+    }
+}
+
+private val promptsSections = listOf(ALL, ANSWERED, NOT_ANSWERED)
+
+enum class PromptSection(@StringRes val titleRes: Int) {
```

Comment:
Good idea 👍

Reply:
> Hey @metalwihen 👋 thank you for the PR!
> 
> When tapping the tabs it seems like the ripple effect is going beyond the tab View (and even overlapping with the Toolbar). Would you please have a look? <img alt="image" width="326" src="https://user-images.githubusercontent.com/14964993/194653704-26f4db17-301d-4fa4-86d4-25d546a241e2.png">

I've found a fix and pushed it. The ripple effect is now contained within the tab row. 
Screenshot: 
![image](https://user-images.githubusercontent.com/2140539/194766496-a2443702-ccf4-4be9-b921-49553f68b00a.png)